src/pages/create/recruitCreatePage/RecruitCreatePage.tsx

```diff
@@ -1,32 +1,30 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import S from './RecruitCreatePage.styled';
-import { TitleAndIntro } from '../../../components/index';
-import RecruitInfoWrapper from './wrappers/RecruitInfoWrapper';
-import RecruitPostWrapper from './wrappers/RecruitPostWrapper';
+import {
+	BasicInformation,
+	ControlButtons,
+	Description,
+	DetailedInformation,
+	RecruitRoles,
+} from '../../../components/index';
 import { useMutation } from '@tanstack/react-query';
-import { useRecoilValue, useRecoilState } from 'recoil';
+import { useRecoilState, useSetRecoilState, useRecoilValue } from 'recoil';
 import { recruitInputState, validState } from '../../../atom';
-import { postingRecruit } from '../../../api';
+import { postingRecruit } from '../../../service/recruit/posting';
 import { useNavigate } from 'react-router-dom';
-
-const descriptions = [
-	'함께할 멤버들에게 알릴 기본 정보들을 기입해주세요!',
-	'기본 정보를 기반으로 구인글을 제공할 수 있습니다.',
-];
-
-const introductions = [
-	'게시될 구인글을 작성해주세요!',
-	'어떤 방식으로 진행하고 싶은지 구체적으로 작성해주세요.',
-];
+import RecruitTags from '../../../components/recruit/create/RecruitTags';
+import { InputState } from '../../../types';
 
 const RecruitCreatePage = () => {
 	const navigate = useNavigate();
-	const recruitFormData = useRecoilValue(recruitInputState);
-	const [isSubmit, setIsSubmit] = useRecoilState(validState);
-	const onClickCancel = () => {
-		// 모달창 띄워서 한 번 더 확인시키고 이동하기
-		// navigate('/');
-	};
+	const setIsSubmit = useSetRecoilState(validState);
+	const [formData, setFormData] = useRecoilState<InputState>(recruitInputState);
+	const validCheck = useRecoilValue(validState);
+
+	const uploadPost = useMutation({
+		mutationFn: (formData: any) => postingRecruit(formData),
+	});
+
```

Comment:
```suggestion
	const uploadPost = useMutation({
		mutationFn: (formData: InputState) => postingRecruit(formData),
		onSuccess: () => {
			navigate(`/recruit/${uploadPost}`);
		},
	});
```

기존에는 전송시에 api 에러가 나도 이동을 하게 되니까, 데이터 전송 성공 후에 페이지 이동하는 로직이 보이도록 리팩토링 해보았습니다!
또한 타입도 any에서 해당 타입으로 변경해보았는데 어떤가요?

Reply:
넵넵!!! 현재 any를 제거해도 되도록 타입 정의를 해놓아서 바로 any를 제거했습니다~
오!! 이렇게 사용할 수 있겠네요!!! 피드백 감사합니다~😊

Reply:
수정했습니다!